### ISSUE TITLE
Nice-to-have: Allow hooking to "pull_request" event instead of "push"

### ISSUE THREAD

**Issue by nkuehn** [vale-cli/vale-action#10]
Since we only need the writing style check when a branch goes into PR review we'd be happy to save some github action runtime contingent by hooking the vale checks only to the `pull_request` github event (which defaults to opening a PR and synchronize, which is when a new head commit is pushed).  That way it would run only on the branches that we need it to run. 

I tried and it started (including logging the number of issues found), but the checks were not annotated to code.  No errors on the logging output, too so I can't provide details here. 

Really just a nice to have but if it works I think it's a better fit to typical workflows.

**Comment by melissamcewen** [comment]
I had the same issue as well. I had to change it to pull to get it to work.

**Comment by jdkato** [comment]
This should be fixed now.

(You may still run into #12, but you'll get the alerts printed to the console at worst.)

**Comment by nkuehn** [comment]
Is there anything to know, e.g. restriction or special config?  I can't find a release or documentation.  

---
PS:  just found this commit: https://github.com/errata-ai/vale-action/commit/a62b0200cda61eb6caa625a8cf5ecd18d779c61b#diff-8cfead41d88ad47d44509a8ab0a109adR23-R28   (assuming it's not a release yet)

**Comment by jdkato** [comment]
The release is [v1.2.0](https://github.com/marketplace/actions/vale-linter) (currently on the marketplace).
